search/src/main/java/artic/edu/search/SearchFragment.kt

```diff
@@ -49,16 +52,16 @@ class SearchFragment : BaseViewModelFragment<SearchViewModel>() {
 
         }
 
-        searchEditText
-                .afterTextChangeEvents()
-                .skipInitialValue()
-                .throttleLast(250, TimeUnit.MILLISECONDS)
-                .subscribe { event ->
-                    event.editable()?.let { editable ->
-                        viewModel.onTextChanged(editable.toString())
-                    }
-                }
-                .disposedBy(disposeBag)
+        bindSearchText()
+
+        viewModel.searchText.subscribe {
```

Comment:
Put a small comment saying `it's important that we set up these observables on the text in this way because otherwise we seriously risk an infinite loop.`